### PR TITLE
RTEMIS-61: Implemented entry_class feature in school_details.

### PR DESCRIPTION
**Pull request by jaswinsingh**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-61

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Implemented entry_class feature in school_details.

### CODE REVIEW

File: modules/rte_mis_school/js/entry_class.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          if ($(this).val() === '_none') {
+            $(this).children('option[value!="_none"]').remove();
+            // Populate select list with values of selected radio buttons
+            var $wrapper = $('#entry-class-wrapper');
+            var $radioButtons = $('#default-entry-class input[type="radio"]:checked, #optional-entry-class input[type="radio"]:checked');
+            // var radioButtons = $('#default-entry-class input[type="radio"]:checked');
+            $radioButtons.each(function() {
+              var radioValue = $(this).val();
+              var radioLabel = $(this).siblings('label').text();
+              console.log();
```
Comment (kunal-sin) [inline modules/rte_mis_school/js/entry_class.js:17]: Remove console log

File: modules/rte_mis_school/js/entry_class.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    attach: function (context, settings) {
+      $(once('entry-class', 'body', context)).each(function () {
+
+        $('#entry-class-wrapper .entry-class-select').each(function() {
+          // Check if the current value is "_none"
+          if ($(this).val() === '_none') {
+            $(this).children('option[value!="_none"]').remove();
+            // Populate select list with values of selected radio buttons
+            var $wrapper = $('#entry-class-wrapper');
+            var $radioButtons = $('#default-entry-class input[type="radio"]:checked, #optional-entry-class input[type="radio"]:checked');
```
Comment (kunal-sin) [inline modules/rte_mis_school/js/entry_class.js:12]: Why we are adding `$` with variable name?

File: modules/rte_mis_school/rte_mis_school.install [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      $field_details['field_label'] = 'Total number of new students of ' . $key . ' to be enrolled in RTE';
+      rte_mis_school_create_integer_field($field_details);
+    }
+  }
+}
+
+/**
+ * Create integer field for entry_class paragraph.
+ */
+function rte_mis_school_create_integer_field($field_details) {
```
Comment (kunal-sin) [inline modules/rte_mis_school/rte_mis_school.install:41]: Type is missing

Comment (kunal-sin) [inline modules/rte_mis_school/rte_mis_school.install:41]: Param missing

File: modules/rte_mis_school/js/entry_class.js (hunk 1 above)
Comment (jaswinsingh) [reply modules/rte_mis_school/js/entry_class.js]: Updated

File: modules/rte_mis_school/js/entry_class.js (hunk 2 above)
Comment (jaswinsingh) [reply modules/rte_mis_school/js/entry_class.js]: Updated

File: modules/rte_mis_school/rte_mis_school.install (hunk 3 above)
Comment (jaswinsingh) [reply modules/rte_mis_school/rte_mis_school.install]: Updated

Comment (jaswinsingh) [reply modules/rte_mis_school/rte_mis_school.install]: Updated

**Review by kunal-sin — APPROVED** [review]

**Pull request merged**